wait/wait.go

```diff
@@ -15,6 +15,7 @@ type Strategy interface {
 
 type StrategyTarget interface {
 	Host(context.Context) (string, error)
+	Ports(ctx context.Context) (nat.PortMap, error)
```

Comment:
I think we are polluting the interface adding this method here:

```
	exitStrategyTarget does not implement StrategyTarget (missing Ports method)
```

Do you foresee another manner to get the ports of the target?

Reply:
Fixed test strategies.